### PR TITLE
Split guides by target group

### LINKED ISSUE #23: Collect user upgrade guide information for ODS 2.0
The purpose of this issue is to collect all information that users need to know (or which would help them understand what is new) when they upgrade their projects from ODS 1.2.0 to 2.0.

### PR DESCRIPTION
**Pull request by michaelsauter**
There are now guides for administrators, contributors and users.

Further, there is a new update guide for users that explains how to
adopt 2.x.

Closes https://github.com/opendevstack/ods-documentation/issues/23.

Addresses https://github.com/opendevstack/ods-documentation/issues/20
partially.

Screenshot of local docs:

Screenshot 2020-01-27 at 10 28 39

### CODE REVIEW

**Review by stefanlack — DISMISSED** [review]
First:  I realy do appreciate the idea of splitting the documentation to three sections _Admin Guide_, _User Guide_ and _Contributer Guide_   

Second: 
Which grants to a user need to execute the steps from "User Guides --> Update to 2.x --> Grant image-puller rights to your *-cd namespace"?

Tested with my non-admin user:

```
PROJECT_ID=tstsla11
oc policy add-role-to-group system:image-puller system:serviceaccounts:$PROJECT_ID-test -n $PROJECT_ID-cd
```

_Error from server (Forbidden): rolebindings.rbac.authorization.k8s.io is forbidden: User "sla1" cannot list rolebindings.rbac.authorization.k8s.io in the namespace "tstsla11-cd": no RBAC policy matched_

**Comment by stefanlack** [comment]
Please see my suggestion pr at https://github.com/BIX-Digital/opendevstack.github.io/pull/1

File: docs/modules/user-guides/pages/update-2x.adoc [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+Run the following, replacing `$PROJECT_ID` with your project ID:
+
+[source,sh]
+----
+oc policy add-role-to-group system:image-puller system:serviceaccounts:$PROJECT_ID-test -n $PROJECT_ID-cd
+oc policy add-role-to-group system:image-puller system:serviceaccounts:$PROJECT_ID-dev -n $PROJECT_ID-cd
+----
+
+For more information, see https://github.com/opendevstack/ods-core/issues/293[ods-core#293].
```
[the remark this answers is not in the document]
Comment (michaelsauter) [reply docs/modules/user-guides/pages/update-2x.adoc:50]: You need to be admin in `$PROJECT_ID-cd`.

**Comment by michaelsauter** [comment]
Thanks for your suggestion. I would still prefer to merge this as-is (maybe with just the fixes to the links), and then we can discuss additional sub-sections. I'd rather not have so many levels.

**Comment by michaelsauter** [comment]
@stefanlack could we please merge this? This provides huge value for users updating from 1.2.x

**Review by stefanlack — APPROVED** [review]
@michaelsauter feel free to ignore my PR and merge yours. 

Please note that I also fixed some broken links in PR via the following commits, that where introduced by your changes: 
https://github.com/BIX-Digital/opendevstack.github.io/pull/1/commits/8be61cf5072b44f2599e6843ef55640cc63638bf

Those are the commits where I have added changes from the history of https://github.com/opendevstack/ods-core/commits/4051409450fb3892d26edd948651d0be6d0854d4/docs/modules/ROOT/pages/getting-started.adoc):

https://github.com/BIX-Digital/opendevstack.github.io/pull/1/commits/a46b237396ee1058c7d3d47703da056356f8370b
https://github.com/BIX-Digital/opendevstack.github.io/pull/1/commits/efd0887ca0c0a8802a6963a7b636c08c8904076c
https://github.com/BIX-Digital/opendevstack.github.io/pull/1/commits/8126753da633d1b9892e107deae5bab83b1a9df6

**Comment by stefanlack** [comment]
It's ok to separate tasks. Maybe we can discuss  about additional sub-sections in other pr.
Links realy should be corrected in my opinion.
Also, I would  like to see that the two sub-sections named _Update to 2.x_  (from User Guides) and _Version Update_ (from Administrator Guide) are named identical, e.g. _Migration Guide_

**Comment by michaelsauter** [comment]
OK. Merged now. Let's follow this up ... I'll see if I can collect the remaining bits in the next days

**Pull request merged**